README.md

```diff
@@ -6,62 +6,30 @@ This repository is for active development of the Azure SDK for C++. For consumer
+- Open a terminal
+  - **Visual Studio:** Open the Developer Command Prompt: **Tools > Commandline > Developer Command Prompt**
+  - **VSCode:** Open a new Terminal in VSCode: **Terminal > New Terminal**
+- Add the `azure-identity-cpp` and `azure-storage-blobs-cpp` libraries with the following command:
 
-##### 1. Create a [CMake](https://cmake.org/cmake/help/latest/) project
-
-CMake will take care of cross-operating system support.
-
-> Visual Studio installs CMake without adding it to the path. You need to [install CMake](https://cmake.org/download/) if you are not using Visual Studio or if you want to use a command line outside Visual Studio.
-
-Visual Studio:
-
-If you are using Visual Studio and you installed [support for CMake](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019), you can create a new CMake Project from Visual Studio, new project menu.
-
--IMAGE HERE Visual Studio-
-
-Visual Studio Code:
-
-Install the VSCode extensions: [CMake](https://marketplace.visualstudio.com/items?itemName=twxs.cmake) and [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools). Then, create folder for your project and open it with VSCode. Press `F1` and type _CMake: Quick Start_, follow the steps to give a name to your project, to select a compiler and any other initial configuration.
-
--IMAGE HERE VSCode-
-
-> You can also manually create the root `CMakeLists.txt` with your own initial configuration and source.
-
-##### 2. Link the Vcpkg toolchain file to your CMake project
-
-Azure SDK provides a CMake module that you can use for your application. You only need to create a folder called _cmake-modules_ on the top level of your CMake project and copy [AzureVcpkg.cmake](https://github.com/Azure/azure-sdk-for-cpp/blob/main/cmake-modules/AzureVcpkg.cmake) to this folder.
-
-The AzureVcpkg module supports three scenarios:
-
-1. Getting and setting up Vcpkg automatically (default case). You can set the env var `AZURE_SDK_DISABLE_AUTO_VCPKG` to disable this behavior.
-2. Automatically linking your application to an existing Vcpkg folder. Set the environment variable `VCPKG_ROOT` to the Vcpkg folder you want to link.
-3. Manually setting a toolchain file with cmake command option. `AzureVcpkg.cmake` module will respect the option.
-
-Add the next lines to your root `CMakeLists.txt` to use `AzureVcpkg.cmake` module:
-
-```cmake
-# Add this lines on the top, before the call to `project(name VERSION 0.0.0)
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules")
-include(AzureVcpkg)
-az_vcpkg_integrate()
+```console
+vcpkg add port azure-identity-cpp azure-storage-blobs-cpp
 ```
 
-##### 3. Add Vcpkg manifest
-
-Add a new file called `vcpkg.json` on the root of your CMake project and add the Azure SDK clients you want to use in your application. For example, the next manifest defines Azure Identity and Blobs.
+- Your `vcpkg.json` should now contain:
 
 ```json
 {
```

Comment:
Since we are dealing with HellowWorld below in the cmakelists I would expect the vcpg.json to also be HelloWorld

Reply:
I removed name and version-string properties as they're not needed in the example and one that gets created in the vcpkg docs scenario doesn't use the properties anyway.